composer.lock

```diff
@@ -0,0 +1,2014 @@
+{
```

Comment:
The composer.lock file shouldn't be committed for libraries. Applications should lock dependencies, but libraries should specify (via composer.json) all the supported dependency versions and leave it up to the implementing app to determine the final selected versions within those bounds.

Reply:
OK. I gonna remove this file.